### PR TITLE
feat: add change camera angle

### PR DESCRIPTION
**Pull request by taibohess**

### CODE REVIEW

**Review by wifimug — APPROVED** [review]

File: src/controllers/camera_controller.py [hunk 1]
```diff
@@ -2,15 +2,26 @@
...
 
 class CameraController:
-    """
-    Setup default camera angle and link it to a collection.
-    """
+    # """
+    # Setup default camera angle and link it to a collection.
+    # """
+    # def setup_camera(self, cam="camera_one", camera_location=(10,5,0),
+    #                  camera_rotation=(1.57057,0.00174533,1.57057), collection_name="Collection"):
```
Comment (DanielSwarup) [inline src/controllers/camera_controller.py:9]: Ideally, do not commit commented-out code.

File: tests/test_launch.py [hunk 2]
```diff
@@ -13,11 +13,11 @@ def setUp(self):
...
-    def tearDown(self):
-        # Clean up test environment
-        os.remove(self.test_file)
-        os.remove(self.expected_output_file)
-        os.remove("tests/expected_output_seg.png")
+    # def tearDown(self):
+    #     # Clean up test environment
+    #     os.remove(self.test_file)
+    #     os.remove(self.expected_output_file)
+    #     os.remove("tests/expected_output_seg.png")
```
Comment (DanielSwarup) [inline tests/test_launch.py:20]: Tear down needs to be uncommented

**Review by DanielSwarup — CHANGES_REQUESTED** [review]

File: src/controllers/camera_controller.py [hunk 3]
```diff
@@ -2,15 +2,26 @@
...
+    #     bpy.data.objects[cam].data.lens_unit = "FOV"
+    #     bpy.data.objects[cam].data.angle = math.radians(100) # distance of camera from the scene
+    #     collection = bpy.data.collections.new(name=collection_name)
+    #     bpy.context.scene.collection.children.link(collection)     # link to the collection containing the crops
+    #     return collection
     def setup_camera(self, cam="camera_one", camera_location=(10,5,0),
                      camera_rotation=(1.57057,0.00174533,1.57057), collection_name="Collection"):
         bpy.ops.object.camera_add(enter_editmode=False, align="VIEW",
                                   location=camera_location, rotation=camera_rotation)
         print("getting to render")
```
Comment (DanielSwarup) [inline src/controllers/camera_controller.py:24]: print statement should be removed

**Review by DanielSwarup — CHANGES_REQUESTED** [review]

**Review by DanielSwarup — CHANGES_REQUESTED** [review]

**Comment by DanielSwarup** [comment]
Small note: When you're done fixing the comments click the "Resolve Conversation" button so it's easier to navigate.

**Review by DanielSwarup — APPROVED** [review]
LGTM!

**Comment by taibohess** [comment]
feat: add change camera angle

**Comment by taibohess** [comment]
feat: add change camera angle

**Pull request closed without merging**